addons/base_import_module/tests/test_cloc.py

```diff
@@ -252,7 +252,38 @@ def test_exclude_cloc_imported_module(self):
         # Import test module
         self.env['ir.module.module']._import_zipfile(stream)
 
+        cl = cloc.Cloc()
+        cl.count_customization(self.env)
+        self.assertEqual(cl.code.get('test_imported_module', 0), 0)
```

Comment:
Why a diff?

Reply:
Seems good now, it was git diff as the test ending is similar